pyaedt/edb_grpc/core/edb_data/nets_data.py

```diff
@@ -0,0 +1,157 @@
+from pyaedt.edb_grpc.core.edb_data.padstacks_data import EDBPadstackInstance
+from pyaedt.edb_grpc.core.edb_data.primitives_data import EDBPrimitives
+from pyaedt.generic.general_methods import pyaedt_function_handler
+
+
+class EDBNetsData(object):
+    """Manages EDB functionalities for a primitives.
+    It Inherits EDB Object properties.
+
+    Examples
+    --------
+    >>> from pyaedt import Edb
+    >>> edb = Edb(myedb, edbversion="2021.2")
+    >>> edb_net = edb.core_nets.nets["GND"]
+    >>> edb_net.name # Class Property
+    >>> edb_net.GetName() # EDB Object Property
+    """
+
+    def __getattr__(self, key):
+        try:
+            return self[key]
+        except:
+            try:
+                return getattr(self.net_object, key)
+            except AttributeError:
+                raise AttributeError("Attribute not present")
+
+    def __init__(self, raw_net, core_app):
+        self._app = core_app
+        self._core_components = core_app.core_components
+        self._core_primitive = core_app.core_primitives
+        self.net_object = raw_net
+
+    @property
+    def name(self):
+        """Return the Net Name.
+
+        Returns
+        -------
+        str
+        """
+        return self.net_object.GetName()
+
+    @name.setter
+    def name(self, val):
+        self.net_object.SetName(val)
+
+    @property
+    def primitives(self):
+        """Return the list of primitives that belongs to the net.
+
+        Returns
+        -------
+        list of :class:`pyaedt.edb_grpc.core.edb_data.primitives_data.EDBPrimitives`
+        """
+        return [EDBPrimitives(i, self._app) for i in self.net_object.Primitives]
+
+    @property
+    def padstack_instances(self):
+        """Return the list of primitives that belongs to the net.
+
+        Returns
+        -------
+        list of :class:`pyaedt.edb_grpc.core.edb_data.padstacks_data.EDBPadstackInstance`"""
+        name = self.name
+        return [
+            EDBPadstackInstance(i, self._app) for i in self.net_object.PadstackInstances if i.GetNet().GetName() == name
+        ]
+
+    @property
+    def is_power_ground(self):
+        """Either to get/set boolean for power/ground net.
```

Comment:
```suggestion
        """Whether the net is power or ground.
```
This original description was too unclear to be certain that my edits are OK. Please check.